UnityProject/Assets/Scripts/Systems/Voting/VotingManager.cs

```diff
@@ -102,18 +126,62 @@ public void TryInitiateRestartVote(GameObject instigator, NetworkConnection send
 		voteType = VoteType.RestartRound;
 		votePolicy = VotePolicy.MajorityRules;
```

Comment:
all these enums are perfect for a strategy pattern https://refactoring.guru/design-patterns/strategy

Reply:
This seems like the modular system i was talking about, but right now i think it's good as it is until someone picks this up again and implements admin voting and other things.

Reply:
you sure you don't want to test implementing it? It is fun!

Reply:
Maybe in an another PR, right now im kinda stuck on this because Unity really hates when i make any prefab changes like this then switch PRs and I want to fix other things like the give item tab for admins.

Reply:
*sad design patterns fanboy noises*